packages/driver/src/adapter.crypto.ts

```diff
@@ -0,0 +1,33 @@
+import type { CryptoUtils } from "./utils";
```

Comment:
Not sure it's needed to have "adapter." prefix in all files that facade one API or another depending on Node vs Browser